src/state/get-drag-impact.js

```diff
@@ -77,6 +77,22 @@ export default ({
 
   const axis: Axis = droppable.axis;
 
+  if (!droppable.isEnabled) {
+    const homeDroppableId = draggingDimension.droppableId;
+    const homeIndex = getDraggablesInsideDroppable(
+      droppables[homeDroppableId],
+      draggables,
+    ).indexOf(draggingDimension);
+    return {
```

Comment:
We need to actually return an impact with a destination or else we get errors when trying to lift from a disabled droppable.

Reply:
perfect

Reply:
can you please add that as a comment in the code?